docs/get-involved/run-node/paratime-node.mdx

```diff
@@ -102,6 +102,15 @@ committee on Mainnet, your entity needs to:
 - Have a validator in the **validator set**.
 - Have at least **5,000,000.00 ROSE staked/delegated** to it.
 
+### Rewards
+
+* The ParaTime will release tokens on-chain to reward nodes for participation.
+* The reward program is 2 years long, and tokens will be released per epoch.
```

Comment:
Why add this now given that this was launched in 2021 and it was 2 years long?

Reply:
Oops, I just made it consistent with Cipher. Is there a new reward schedule for running Emerald/Sapphire/Cipher compute nodes?

Reply:
I would remove all of these for now.

Reply:
Removed.